apps/daimo-mobile/src/view/screen/onboarding/IntroPages.tsx

```diff
@@ -0,0 +1,175 @@
+import { ReactNode, useState } from "react";
+import {
+  NativeScrollEvent,
+  NativeSyntheticEvent,
+  ScrollView,
+  View,
+  StyleSheet,
+} from "react-native";
+
+import { ButtonBig, TextButton } from "../../shared/Button";
+import { InfoLink } from "../../shared/InfoLink";
+import { IntroTextParagraph } from "../../shared/IntroTextParagraph";
+import Spacer from "../../shared/Spacer";
+import { color, ss } from "../../shared/style";
+import { TextCenter, TextH1 } from "../../shared/text";
+
+export function IntroPages({
+  onNext,
+}: {
+  onNext: ({ choice }: { choice: "create" | "existing" }) => void;
+}) {
+  const [pageIndex, setPageIndex] = useState(0);
+  const updatePageBubble = (event: NativeSyntheticEvent<NativeScrollEvent>) => {
+    const { contentOffset, layoutMeasurement } = event.nativeEvent;
+    const page = Math.round(contentOffset.x / layoutMeasurement.width);
+    setPageIndex(page);
+  };
+
+  return (
+    <View style={styles.onboardingPage}>
+      <View style={styles.introPages}>
+        <PageBubble count={4} index={pageIndex} />
+        <ScrollView
+          horizontal
+          pagingEnabled
+          showsHorizontalScrollIndicator={false}
+          style={styles.introPageScroll}
+          onScroll={updatePageBubble}
+          scrollEventThrottle={32}
+          contentContainerStyle={{ width: `${introPages.length * 100}%` }}
+        >
+          {introPages.map((page, i) => (
+            <View style={{ width: `${100 / introPages.length}%` }} key={i}>
+              {page}
+            </View>
+          ))}
+        </ScrollView>
+        <Spacer h={32} />
+        <View style={styles.introButtonsCenter}>
+          <View style={styles.introButtonsWrap}>
+            <ButtonBig
+              type="primary"
+              title="Create Account"
+              onPress={() => {
+                onNext({ choice: "create" });
+              }}
+            />
+            <Spacer h={16} />
+            <TextButton
+              title="Already have an account?"
+              onPress={() => {
+                onNext({ choice: "existing" });
+              }}
+            />
+          </View>
+        </View>
+      </View>
+    </View>
+  );
+}
+
+const tokenSymbol = "USDC";
+const introPages = [
+  <IntroPage title="Welcome to Daimo">
+    <IntroTextParagraph>
+      Daimo is a global payments app that runs on Ethereum. Send and receive
+      USDC on Base mainnet.
```

Comment:
maybe delete second line? (i know it was there before)

Reply:
done